pkgs/by-name/c9/c99sh/package.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  patsh,
+  pkg-config,
+  bashNonInteractive,
+  targetPackages,
+  cc ? targetPackages.stdenv.cc,
+}:
+
+stdenv.mkDerivation {
+  pname = "c99sh";
+  version = "1.1.0-unstable-2024-02-12";
+
+  src = fetchFromGitHub {
+    owner = "RhysU";
+    repo = "c99sh";
+    rev = "f01d549c38878561d27bd1112ed49ccf7211a85c";
+    hash = "sha256-jnmShHhkJ4eusyXH7CMr1mwboXnkn+zjCn2E3q/XXy4=";
+  };
+
+  nativeBuildInputs = [ patsh ];
+
+  buildInputs = [
+    pkg-config
+    bashNonInteractive # FIXME: patchShebangs doesn't function in cross-builds
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir --parents $out/bin/
+    cp --no-dereference c*sh $out/bin # c11sh and cxxsh
+    patsh -f $out/bin/c99sh -s ${builtins.storeDir} --path "$HOST_PATH"
+    substituteInPlace $out/bin/c99sh \
+      --replace-fail -cc -${lib.getBin cc}/bin/cc \
+      --replace-fail -c++ -${lib.getBin cc}/bin/c++
+
+    runHook postInstall
+  '';
+
+  meta = {
+    description = ''
+      A shebang-friendly script for "interpreting" single C99, C11,
+      and C++ files, including rcfile support.
+    '';
```

Comment:
This desscription should be single line and should not start with an article or end with punctuation